src/DependencyInjection/Compiler/TweakCompilerPass.php

```diff
@@ -116,4 +136,18 @@ private function replaceBlockName(ContainerBuilder $container, Definition $defin
             );
         }
     }
+
+    /**
+     * @return string[]
```

Comment:
Are you sure? When reading the code, I'm not under the impression that `$attribute` is a string.

Reply:
Yes, it's a string, if you declate a `<tag ... context="foo">` inside your `services.xml`

Reply:
No, the method should return just a string array

Reply:
Ooh right my bad, I thought this was an `@param` phpdoc! Can you please add it?

Reply:
```suggestion
     * @param string[][]
     *
     * @return string[]
```